compiler/src/dmd/cxxfrontend.d

```diff
@@ -547,6 +547,12 @@ Type addMod(Type type, MOD mod)
     return dmd.typesem.addMod(type, mod);
 }
 
+Type addStorageClass(Type type, StorageClass stc)
+{
+    import dmd.typesem;
+    return dmd.typesem.addStorageClass(type, stc);
```

Comment:
It would be good if there was a test for this in cxxfrontend.cc, I already started this off by adding a link test for all functions under dmd namespace.